src/sass/partials/components/_content-card.scss

```diff
@@ -0,0 +1,30 @@
+.card {
+  border: setting-get('input border width') solid setting-get('secondary color');
+  display: flex;
+  flex-direction: column;
+  margin-top: (setting-get('rhythm') / .5);
+  padding: setting-get('rhythm') / .75;
+
+  &--link {
+    @include link(setting-get('active link color'), setting-get('active link color'), setting-get('secondary link color'), setting-get('secondary link color'));
```

Comment:
Just realized that I'm not actually using this class for this component, but I will still make the `base-link` mixin change.